### PR TITLE
[Backend/Admin/CustomerSupport] - Create CustomerSupport Controller

### PR DESCRIPTION
**Pull request by shun2134**
・Create Controller
-  Http > Controllers > Inquiries > CustomerSupportController.php

・Connect each Models
- Inquiry.php belongs To InquiryGenre / InquiryStatus / Customer

・Modify blade.php 
- views > admin > inquiry > customerSupport.blade.php : create route and variable data
- views > admin > inquiry > modal > customerStatus.blade.php :create route and variable data 
- views > admin > inquiry > modal > translateStatus.blade.php : create route and variable data

・Add route on web.php

### CODE REVIEW

File: app/Http/Controllers/Inquiries/CustomerSupportController.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+class CustomerSupportController extends Controller
+{
+    private $inquiry;
+    private $inquiry_genre;
+    private $inquiry_status;
+
+    public function __construct(Inquiry $inquiry, InquiryGenre $inquiry_genre, InquiryStatus $inquiry_status)
+    {
+        $this->inquiry        = $inquiry;
```
Comment (kredo-school) [inline app/Http/Controllers/Inquiries/CustomerSupportController.php]: avoid too much spacing

File: app/Http/Controllers/Inquiries/CustomerSupportController.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    }
+
+    public function update(Request $request, $id)
+    {
+        DB::beginTransaction();
+        
+        // Validate the incoming request data (optional)
+        // $request->validate([
+        //     'answer'               => 'required|string|max:255',
+        //     'translated_answer'    => 'required|string|max:255',
```
Comment (kredo-school) [inline app/Http/Controllers/Inquiries/CustomerSupportController.php]: remove commented codes

File: app/Http/Controllers/Inquiries/CustomerSupportController.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        ]);
+
+
+        try {
+            // Find the admin record by its ID
+            $inquiry = Inquiry::findOrFail($id);
+            
+            // Update the admin record with the new data
+            // $inquiry->answer               = $request->input('answer');
+            // $inquiry->translated_answer    = $request->input('translated_answer');
```
Comment (kredo-school) [inline app/Http/Controllers/Inquiries/CustomerSupportController.php]: remove commented codes

File: app/Models/Inquiries/Inquiry.php [hunk 4]
```diff
@@ -2,12 +2,50 @@
...
+        'translated_answer',
+        'customer_id',
+        'genre_id',
+        'inquiry_status_id',
+        'status'
+    ];
+
+    public function getData()
+    {
+        $data = DB::table($this->table)->get('id', 'title', 'content', 'answer', 'translated_answer', 'customer_id', 'genre_id', 'inquiry_status_id');
```
Comment (kredo-school) [inline app/Models/Inquiries/Inquiry.php]: as much as possible use the eloquent way:

```
public static function getData()
{
    return self::select('id', 'title', 'content', 'answer', 'translated_answer', 'customer_id', 'genre_id', 'inquiry_status_id')->get();
}
```

**Comment by shun2134** [comment]
I fixed "CustomerSupport.blade.php" and "Inquiry.php".

**Pull request merged**